### DISCUSSION TITLE
Collection and Study Managment

### DISCUSSION THREAD

**Discussion by diaabadr** [camicroscope/GSOC#16] [Q&A] [answered]
Hi @birm 
Collection button doesn't work. Does it's functionality added or not yet? please i need more details on the idea (collection and study management) requirements if you can provide me with additional info. 
![image](https://user-images.githubusercontent.com/77173710/224565142-bc6af1b2-777a-4db7-ac89-53a049273d55.png)

**Comment by birm** [comment, marked as answer]
I'm going to tag @nanli-emory in this one.

**Reply by nanli-emory** [reply]
Hi @diaabadr, sorry for delaying in response. Have you tried to figure out why the collection button doesn't work?

**Reply by himanshu1409** [reply]
Referring to the codebase of caMicroscope 
<img width="818" alt="Screenshot 2023-03-30 at 9 09 50 PM" src="https://user-images.githubusercontent.com/120622494/228890724-c18f8e10-1fab-43f6-b8e7-f3078ad70e41.png">
the Collection button lies inside a div container whereas it should have been inside a button tag like the Reload button. Adding an onclick() function would add desired functionality to it.
